### PR TITLE
Cleanup LinearSolversApplication

### PR DESCRIPTION
**Pull request by oberbichler**
**Description**
Renaming of files and names from `EigenSolvers...` to `LinearSolvers...` after #7182 

**Changelog**
- Simple search & replace of all remaining `EigenSolvers...`
- Rename files containing `EigenSolvers...`
- Unify ASCII-Logos

### CODE REVIEW

File: applications/LinearSolversApplication/custom_python/add_custom_solvers_to_python.cpp [hunk 1]
```diff
@@ -41,7 +39,7 @@
 #endif
 
 #include "factories/standard_linear_solver_factory.h"
-#include "eigen_solvers_application.h"
+#include "linear_solvers_application.h"
```
Comment (philbucher) [inline applications/LinearSolversApplication/custom_python/add_custom_solvers_to_python.cpp]: I don't think this include should be here, can you check please?

**Review by philbucher — COMMENTED** [review]
one comment otherwise looks good, thx for the cleanup

**Review by philbucher — APPROVED** [review]
if it compiles it means its ready :)

Thx again for the effort!

**Comment by oberbichler** [comment]
with pleasure ;) just wondering about the codacy error. the other applications are also using the star-import.

**Comment by philbucher** [comment]
> with pleasure ;) just wondering about the codacy error. the other applications are also using the star-import.

don't worry abt this warning, this cannot be avoided
The C++ stuff (kinda has to) is imported with star

**Pull request merged**